src/Aquifer.Public.API/Endpoints/Bibles/Alignments/Greek/Request.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.ComponentModel;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.Alignments.Greek;
+
+public class Request
+{
+    /// <summary>
+    /// The id of the Bible for which to return text.
+    /// </summary>
+    public int BibleId { get; init; }
+
+    /// <summary>
+    /// Book code based off USFM book identifier (e.g. GEN, EXO, etc.). Can get a list of available books and identifiers
+    /// from the /bibles/books endpoint. This parameter is required.
+    /// </summary>
+    public string BookCode { get; init; } = null!;
+
+    /// <summary>
+    /// Optional start chapter in the book.
+    /// </summary>
+    [DefaultValue(1)]
```

Comment:
This kinda goes back to earlier discussions, but I'd prefer not having the attribute here since that's what is used to generate the OpenAPI spec and it feels like it's exposing our internal implementation choices by having them (especially the `999` values below). If we remove the attribute it should keep working as it does now but make the API docs look cleaner. Thoughts?

Reply:
@MrGrinst, this is a byproduct of not using nullable for these start/end request parameters.  @jwinston-bn asked for non-nullable on the GET Bible Texts route with defaults of `1` and `999` for start and end values, respectively.  I noticed later that doing this prevented the documentation page from showing these values as "optional."  We were able to address that by specifying `DefaultValueAttribute` because [our documentation creation code](https://github.com/BiblioNexusStudio/aquifer-server/blob/214ca06cd5a409bb243c8db856433037c93eda7d/src/Aquifer.Public.API/OpenApi/DefaultParameterOperationProcessor.cs#L34-L38) checks for that attribute and  lists the parameter as optional (it also seems that the NSwag itself also checks for nullable types and makes them optional) .  Here's the [original fix](https://github.com/BiblioNexusStudio/aquifer-server/pull/450) for that route.  I then copied the same request parameters to this Greek alignment text route for consistency across the Public API surface.

We _could_ specify `DefaultValue(null)` instead (which @jwinston-bn pointed out in a Slack DM to me).  My response from that Slack thread:

> I agree that `DefaultValue(null)` has the best result for the consumer in this case.  However, I don't like the potential confusion for future developers because that's not actually the default value.

> There's a `RequiredAttribute` (https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=net-8.0) from the same namespace as `DefaultValueAttribute` that we could also reference from the `DefaultParameterOperationProcessor` code but that would likely require slapping that `Required` attribute on all required properties and using it exclusively.  It would be a bit less magical than the current implementation because it makes it more clear why that text appears.

> There might be danger of that `RequiredAttribute` causing deserialization headaches so we could also just make our own `OptionalAttribute` and use that in addition to `DefaultValue` to determine if the "required" text should show.

We eventually decided to table it and just ship with the actual default value in the attribute.

All that said, my preferred solution would be nullable request properties when the route structure allows them to be optional, possibly with the addition of a `RequiredAttribute` for specifying query string parameters that we require for the route to function.  Then we'd get the correct documentation for optional properties without needing an attribute and we could keep the actual default value in our internal implementation.  We'd also know when the user isn't supplying values for validation purposes which helps solve edge cases.